milestone-3/frontend/src/components/recipepage/recipepage.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import "./recipepage.css";
+
+interface recipePageProps {
+  name: string;
+  desc: string;
+  image: string;
+  ingred: string[];
+  instruc: string[];
+}
```

Comment:
this recipe page props is the exact same type as the Recipe interface you have in your recipeData.ts. Once you move that Recipe interface to some sort of types.ts file then you can just import that interface and use it here, as opposed to redeclaring several interfaces with the same type